apps/web/layouts/main.tsx

```diff
@@ -45,7 +45,7 @@ import { MenuTabsContext } from '../context/MenuTabsContext/MenuTabsContext'
 import { useI18n } from '../i18n'
 import { GET_ALERT_BANNER_QUERY } from '../screens/queries/AlertBanner'
 import { environment } from '../environments'
-import { useNamespace } from '../hooks'
+import { useFeatureFlag, useNamespace } from '../hooks'
```

Comment:
Isn't this available in a module?

Reply:
This is the specific hook for the web project which behaves differently from the global useFeatureFlag hook, if that's what you are thinking about.

Reply:
Or perhaps you are talking about the import being @island.is/web/components instead of this relative path :)